templates/prometheus/010-ConfigMap-prometheus-config.yaml

```diff
@@ -122,10 +122,18 @@ data:
           severity: warning
         expr: kube_pod_container_status_terminated == 1
         for: 60s
-      - alert: Restarting Pods
+      - alert: Restarting_Pods
         annotations:
           description: '{{ $labels.pod }}'
           value: "{{ $value }}"
           severity: warning
         expr: rate(kube_pod_container_status_restarts_total[10m]) * 600 > 5
         for: 60s
+      - alert: Pending_Pods
+        annotations:
+          description: '{{ $labels.pod }}'
+          phase: '{{ $labels.phase }}'
+          value: "{{ $value }}"
+          severity: warning
+        expr: kube_pod_status_phase{phase="Pending"} == 1
+        for: 60s
```

Comment:
Suggest bumping these alerts to 180 seconds or something. At least for pending, it can be due to long container image pull time

Reply:
Good point. Done!